src/components/navbar/navDropdown/NavDropdown.module.scss

```diff
@@ -0,0 +1,83 @@
+@import '../../../utils/variables';
+.dropdownContainer.active {
+    transform: translate3d(0, 0, 0);
+    visibility: 'visible';
+    // opacity: 1;
+    // background-color: palevioletred;
+}
+.dropdownContainer {
+    z-index: 1;
+    background-color: $white;
+    position: absolute;
+    display: inherit;
+    top: 0;
+    left: calc(5vw - 10px);
+    width: 90vw;
+    margin: 0 10px 10px 10px;
+    padding: 15px;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    box-shadow: 0px 4px 6px 5px rgba(0, 0, 0, 0.1);
+
+    transform: translate3d(0, -110%, 0);
+    visibility: 'hidden';
+    // opacity: 1;
+    transition: all 0.5s ease-out;
+    // left: 0;
+    // .active {
+    //     transform: translate3d(0, 0, 0);
+    //     visibility: 'visible';
+    //     opacity: 1;
+    //     background-color: palevioletred;
+    // }
+
+    @media (min-width: $medium) {
+        display: none;
+    }
+    @media (max-height: 500px) {
+        height: 100vh;
+        width: 100vw;
+        padding: 5px;
+        margin: 0;
+        left: 0;
+
+        overflow: scroll;
+    }
+
+    header {
+        width: 100%;
+        display: flex;
+        justify-content: space-between;
+        .mobileLogoContainer {
+            position: initial;
+
+            height: auto;
+            display: flex;
+            margin: 0 10px;
+
+            max-width: 250px;
+            width: 100%;
+        }
+        .close {
+            cursor: pointer;
+        }
+    }
+
+    // mobile links are not working, drop down nav, maybe an issue when i removed the z-indexes or find a better way to tell if you clicked on the nav section dropdown
+
+    .mobileLinksList {
+        padding: 0;
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+    }
+}
+.dropdownContainer.active {
+    // transform: translate3d(0, 0, 0);
+    // visibility: 'visible';
+    // opacity: 1;
+    // background-color: palegreen;
+}
```

Comment:
Do we need this?
```suggestion
.dropdownContainer.active {
}
```

Reply:
Nope, sorry. That was me learning how to add specificity to an element without an ID :] @ArronAult suggested I use classNames, it was an experience :]